app/controllers/concerns/blacklight/suggest/suggest_helper.rb

```diff
@@ -0,0 +1,32 @@
+module Blacklight
+  module Suggest
+    module SuggestHelper
+      extend ActiveSupport::Concern
+      include Blacklight::SearchHelper
+
+      ##
+      # For now, only use the q parameter to create a
+      # Blacklight::Suggest::Response
+      # @param [Hash] params
+      # @return [Blacklight::Suggest::Response]
+      def suggestions(params)
+        request_params = { q: params[:q] }
+        Blacklight::Suggest::Response.new suggest_results(request_params), request_params, suggest_handler_path
+      end
+
+      ##
+      # Query the suggest handler using RSolr::Client::send_and_receive
+      # @param [Hash] request_params
+      # @return [RSolr::HashWithResponse]
+      def suggest_results(request_params)
+        repository.connection.send_and_receive(suggest_handler_path, params: request_params)
```

Comment:
Do suggestions bypass any access control mechanisms? I'd guess they do, because they're repository wide, but want to make sure (and maybe we should document that as part of the configuration?)...